lib/modules/manager/gradle/extract/consistent-versions-plugin.ts

```diff
@@ -9,7 +9,7 @@ import { isDependencyString, versionLikeSubstring } from '../utils';
 export const VERSIONS_PROPS = 'versions.props';
 export const VERSIONS_LOCK = 'versions.lock';
 export const LOCKFIlE_HEADER_TEXT = regEx(
-  /^# Run \.\/gradlew (?:--write-locks|writeVersionsLock) to regenerate this file/,
+  /^# Run \.\/gradlew (?:--write-locks|writeVersionsLock|writeVersionsLocks) to regenerate this file/,
```

Comment:
```suggestion
  /^# Run \.\/gradlew (?:--write-locks|writeVersionsLocks?) to regenerate this file/,
```
wouldn't this be simpler?